src/components/UkLocal.vue

```diff
@@ -0,0 +1,111 @@
+<template>
+  <div>
+    <UkRegionSection :NationData="nation"></UkRegionSection>
+    <NearbyCasesFinder
+      :regionData="utla"
+      currentCountry="UK"
+    ></NearbyCasesFinder>
+  </div>
+</template>
+
+<script>
+import UkRegionSection from "./UkRegionSection.vue";
+import NearbyCasesFinder from "./NearbyCasesFinder.vue";
+
+export default {
+  name: "UkLocal",
+  data() {
+    return {
+      nation: null,
+      utla: null,
+      display: null,
+      chartData: null,
+      overview: null,
+    };
+  },
+  async mounted() {
+    const queries = `{
+        "date":"date",
+        "areaName":"areaName",
+        "areaCode":"areaCode",
+        "admissions":"cumAdmissions",
+        "admissionsNew":"newAdmissions",
+        "hospitalCases":"hospitalCases",
+        "covidOccupiedMVBeds":"covidOccupiedMVBeds",
+        "confirmedRate":"cumCasesByPublishDateRate",
+        "confirmedNew":"newCasesByPublishDate",
+        "confirmedNewBySpecimen":"newCasesBySpecimenDate",
+        "confirmed":"cumCasesByPublishDate",
+        "confirmedBySpecimen":"cumCasesBySpecimenDate",
+        "deathNewBySpecimen":"newDeaths28DaysByDeathDate",
+        "deathNew":"newDeaths28DaysByPublishDate",
+        "death":"cumDeaths28DaysByPublishDate",
+        "deathRate":"cumDeaths28DaysByPublishDateRate",
+        "testedNew":"newTestsByPublishDate",
+        "tested":"cumTestsByPublishDate"
+    }`
+      .replaceAll(" ", "")
+      .replaceAll("\n", "");
```

Comment:
amazing huh

Reply:
Lol you should see how amazing [this api](https://coronavirus.data.gov.uk/details/developers-guide) is